actors/builtin/storage_miner/storage_miner_actor.go

```diff
@@ -101,7 +140,7 @@ func (a *StorageMinerActor) _rtCommitWorkerKeyChange(rt Runtime) *adt.EmptyValue
 //////////////////
 
 // Called by StoragePowerActor to notify StorageMiner of SurprisePoSt Challenge.
-func (a *StorageMinerActor) OnSurprisePoStChallenge(rt Runtime) *adt.EmptyValue {
+func (a *StorageMinerActor) OnSurprisePoStChallenge(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
```

Comment:
Similar changes as in #73 https://github.com/filecoin-project/specs-actors/pull/73/files#r372679837